### PR TITLE
Feat (l18n): define expected means of doing l18n (closes #323)

### LINKED ISSUE #323: Add note about i18n
The note should include: 

 * [x] The spec should details how we expect developers to localize content. Like by: `<link rel=manifest href='manifest?lang=en'>`
 * [x]  An example showing the use of a language other than English. 
 * [x] Add discussion of obtaining a correctly localized reference, particularly for use in cases where the source page itself has used language negotiation.

### PR DESCRIPTION
**Pull request by marcoscaceres**
@r12a, would appreciate a review (see #323 for more context).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3016,11 +3016,42 @@ <h3 id="applying">
...
+          language is and dynamically adding or replacing the manifest link
+          relationship to the document based on that language preference (e.g.,
+          using a URL like "manifest?lang=fr").
+        </dd>
+        <dt>
+          Using content-negotiation, or geo-ip, etc. on the server:
+        </dt>
+        <dd>
+          The server that hosts the web application could attempt to
+          predetermine the user's language by using a geo-ip look-up or by
```
Comment (kenchris) [inline index.html]: is geo-ip an established term?

File: index.html [hunk 2]
```diff
@@ -3016,11 +3016,42 @@ <h3 id="applying">
...
+          using content negotiation (e.g., using [[RFC7540]]'s
+          "<code>Accept-Language</code>" header, or even a custom HTTP header).
+        </dd>
+      </dl>
+      <p>
+        Given the options above, developers need to be mindful of the
+        end-user's privacy with respect to their preferred language. Sending
+        the user's preferred language in the clear is generally not ok, as it
```
Comment (kenchris) [inline index.html]: O.K. ? or OK?

File: index.html [hunk 3]
```diff
@@ -3016,11 +3016,42 @@ <h3 id="applying">
...
+        reveals personal information about an end-user. As such, developers are
```
Comment (kenchris) [inline index.html]: the end-user... you use "the" before in the sentence

File: index.html [hunk 4]
```diff
@@ -3016,11 +3016,42 @@ <h3 id="applying">
...
+        encouraged to use [[TLS]] to reduce the chances of pervasive monitoring
+        of their Web applications [[RFC7258]].
```
Comment (mounirlamouri) [inline index.html]: UA send the Accept-Languages header for HTTP requests. That point sounds fairly moot.

File: index.html (hunk 1 above)
Comment (marcoscaceres) [inline index.html]: I could change to Geotargeting and link to https://en.wikipedia.org/wiki/Geotargeting

File: index.html (hunk 4 above)
Comment (marcoscaceres) [inline index.html]: I'm going to try to clarify that it's for when the user explicitly provides the language (i.e., not applicable so much for Accept-Languages, as those are usually just pre-configured in the UA).

File: index.html (hunk 3 above)
Comment (marcoscaceres) [inline index.html]: fixed

File: index.html (hunk 2 above)
Comment (marcoscaceres) [inline index.html]: Fixed.

**Comment by marcoscaceres** [comment]
Will wait a bit for @r12a to comment. He is probably busy at the W3C AC meeting.

**Comment by marcoscaceres** [comment]
Merging this in to keep things moving... @r12a, feel free to do a post merge review. I can address any feedback as separate bugs.

**Pull request merged**